cryptofeed/exchange/probit.py

```diff
@@ -165,7 +166,7 @@ async def message_handler(self, msg: str, timestamp: float):
             await self._l2_update(msg, timestamp)
         # Probit has a 'ticker' channel, but it provide OHLC-last data, not BBO px.
 
-    async def subscribe(self, websocket):
+    async def subscribe(self, websocket: websockets.WebSocketClientProtocol) -> bool:
```

Comment:
why are the subscribe methods all returning a bool?

Reply:
Because some exchanges do not support all channels as websocket subscription.
Therefore some WebsocketFeed poll REST endpoints to retrieve data for some channels.
I am searching an example...